core/src/main/java/tc/oc/pgm/PGMPlugin.java

```diff
@@ -274,17 +274,40 @@ public void reloadConfig() {
     logger.setLevel(config.getLogLevel());
 
     for (String source : config.getMapSources()) {
-      final MapSourceFactory factory;
+      MapSourceFactory factory = null;
       try {
-        factory =
-            source.equalsIgnoreCase("default")
-                ? DefaultMapSourceFactory.INSTANCE
-                : new SystemMapSourceFactory(source);
+        if (source.equalsIgnoreCase("default"))
+          factory =
+              new GitMapSourceFactory(
+                  "https://github.com/KingOfSquares/DefaultMaps", logger, false);
```

Comment:
I don't think having hardcoded values pointing to a personal repo is a good idea. This should be in the config somewhere?

Reply:
> I don't think having hardcoded values pointing to a personal repo is a good idea. This should be in the config somewhere?

Of course, just had to put something there for review.



> Don't define it here. Define it in the default `config.yml`
> We'll change the actual repo before merging.

ok!